draft-ietf-tls-exported-authenticator.md

```diff
@@ -511,6 +511,13 @@ The signatures generated with this API cover the context string
 "Exported Authenticator" and therefore cannot be transplanted into other
 protocols.
 
+In TLS 1.3 the client and server are not guaranteed to agree on the client’s
+final flight until the first application message.  Because EAs can be
+negotiated out-of-band it is possible to negotiate EAs without agreeing on the
+entire transcript.  Servers SHOULD send application data before sending a
+CertificateRequest to the client.  If there is no application data to send the
+server MAY send a NewSessionTicket.
```

Comment:
NewSessionTicket is not an application message, so the previous text does not actually say that receiving NST would suffice to produce full-transcript agreement.  (It would, of course, since NST is protected by application traffic keys, but we should probably tweak some text to be more precise.)

Reply:
Suggest adding "(because that is also protected by application traffic keys)."